x11-libs/vte/vte-0.76.3.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit gnome.org meson python-any-r1 vala xdg
+
+DESCRIPTION="Library providing a virtual terminal emulator widget"
+HOMEPAGE="https://wiki.gnome.org/Apps/Terminal/VTE"
+
+# Upstream is hostile and refuses to upload tarballs.
+SRC_URI="https://gitlab.gnome.org/GNOME/${PN}/-/archive/${PV}/${P}.tar.bz2"
+SRC_URI="${SRC_URI} !vanilla? ( https://dev.gentoo.org/~mattst88/distfiles/${PN}-0.74.0-command-notify.patch.xz )"
+# Once SIXEL support ships (0.66 or later), might need xterm license (but code might be considered upgraded to LGPL-3+)
+LICENSE="LGPL-3+ GPL-3+"
+
+SLOT="2.91"      # vte_api_version in meson.build
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="+crypt debug gtk-doc +icu +introspection systemd +vala vanilla"
+REQUIRED_USE="
+	gtk-doc? ( introspection )
+	vala? ( introspection )
+"
+
+DEPEND="
+	>=x11-libs/gtk+-3.24.22:3[introspection?]
+	>=x11-libs/cairo-1.0
+	>=dev-libs/fribidi-1.0.0
+	>=dev-libs/glib-2.72:2
+	crypt?  ( >=net-libs/gnutls-3.2.7:0= )
+	icu? ( dev-libs/icu:= )
+	>=x11-libs/pango-1.22.0
+	>=dev-libs/libpcre2-10.21:=
+	systemd? ( >=sys-apps/systemd-220:= )
+	>=app-arch/lz4-1.9
+	introspection? ( >=dev-libs/gobject-introspection-1.56:= )
+	x11-libs/pango[introspection?]
+"
+RDEPEND="${DEPEND}
+	~gui-libs/vte-common-${PV}[systemd?]
+"
+BDEPEND="
+	${PYTHON_DEPS}
+	dev-libs/libxml2:2
+	dev-util/glib-utils
+	gtk-doc? ( dev-util/gi-docgen )
+	>=sys-devel/gettext-0.19.8
+	virtual/pkgconfig
+	vala? ( $(vala_depend) )
+"
+
+src_prepare() {
+	default
+	use vala && vala_setup
+	xdg_environment_reset
+
+	use elibc_musl && eapply "${FILESDIR}"/${PN}-0.66.2-musl-W_EXITCODE.patch
+
+	if ! use vanilla; then
```

Comment:
It seems we could use the same patches as got the gtk4 version, but passing the configure options for gtk3:
https://src.fedoraproject.org/rpms/vte291/blob/f40/f/0001-a11y-implement-GtkAccessibleText.patch
https://src.fedoraproject.org/rpms/vte291/blob/f40/f/0001-add-notification-and-shell-precmd-preexec.patch

Reply:
Done in https://github.com/gentoo/gentoo/pull/34844/commits/3642d1ee1c6b354f4e24275fe3104cfe03acd33a